lib/funky/html/parser.rb

```diff
@@ -23,8 +23,8 @@ def extract_views_from(html)
       end
 
       def extract_likes_from(html)
-        html.match /"likecount":(.*?),/
-        matched_count $1
+        likes = html.to_enum(:scan, /"likecount":(.*?),/).map { Regexp.last_match }.last
```

Comment:
This allows me to get the full match data, although I could also just use scan....